keras_core/export/export_lib.py

```diff
@@ -0,0 +1,570 @@
+"""Library for exporting inference-only Keras models/layers."""
+
+import tensorflow as tf
+
+from keras_core import backend
+from keras_core.api_export import keras_core_export
+from keras_core.layers import Layer
+from keras_core.models import Functional
+from keras_core.models import Sequential
+from keras_core.utils import io_utils
+
+
+@keras_core_export("keras_core.export.ExportArchive")
+class ExportArchive(tf.__internal__.tracking.AutoTrackable):
```

Comment:
Since it is fundamentally SavedModel specific I wonder if we should rename it to something that makes it explicit.

Alternatively we can have 1. backend-agnostic `ExportArchive` used for configuring endpoints, a `TFSavedModelExportArchive` subclass that is TF specific.

Reply:
I think we can rename it once we either have a backend-agnostic abstraction or decide how we want to tie in the JAX ExportArchive. Let's discuss further on chat after this.

Reply:
Is the class even reusable with something other than SavedModel in the future? It seems pretty baked into the APIs (e.g. TensorSpecs, tf.function)